libyul/Dialect.h

```diff
@@ -51,6 +51,11 @@ struct BuiltinFunction
 	bool movable = false;
 	/// If true, a call to this function can be omitted without changing semantics.
 	bool sideEffectFree = false;
+	/// If true, a call to this function can be omitted without changing semantics if the
+	/// program does not contain the msize instruction.
+	bool sideEffectFreeUpToMSize = false;
```

Comment:
I'm reading this variable as `sideEffectFreeIfNoMSize` not sure if it's a more readable name :-)

Reply:
Yes, maybe better.